Client/metrics.yaml

```diff
@@ -738,6 +738,28 @@ onboarding:
     notification_emails:
       - fx-ios-data-stewards@mozilla.com
     expires: "2023-07-01"
+  engagement_notification_tapped:
+    type: event
+    description: |
+      Records the users interaction with the engagement notification.
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/?
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/?
+    notification_emails:
+      - fx-ios-data-stewards@mozilla.com
+    expires: "2023-09-01"
+  engagement_notification_cancel:
+    type: event
+    description: |
+      Records the cancelation of a scheduled engagement notification.
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/?
```

Comment:
We need to fill out the pull request number in these links

Reply:
Fixed it [here](https://github.com/mozilla-mobile/firefox-ios/pull/13697).